src/extension.ts

```diff
@@ -27,28 +29,50 @@ let mdbExtension: MDBExtensionController;
 export async function activate(
   context: vscode.ExtensionContext
 ): Promise<void> {
-  log.info('Activating extension...');
   ext.context = context;
+  let hasKeytar = false;
 
   try {
     ext.keytarModule = createKeytar();
+    hasKeytar = true;
   } catch (err) {
     // Couldn't load keytar, proceed without storing & loading connections.
   }
 
+  const defaultConnectionSavingLocation = vscode.workspace
+    .getConfiguration('mdb.connectionSaving')
+    .get('defaultConnectionSavingLocation');
+
+  log.info('Activating extension...', {
```

Comment:
I'm thinking a test for this log might be a very useful one for debugging the environment. Not needed for this pr if we don't want to put the time in now, just a thought.

Reply:
What would you test here? Do we have tests for logs already somewhere in compass?